clio/tpl/ui-project/angular.json

```diff
@@ -38,7 +34,8 @@
 						],
 						"scripts": [],
 						"extraWebpackConfig": "webpack.config.js",
```

Comment:
@besgit clio does not contain template for
webpack.prod.config.js, where is this coming form ?